### ISSUE TITLE
Semantic coloring over-uses orange for types in certain languages

### ISSUE THREAD

**Issue by simurai** [primer/github-vscode-theme#118]
IDK, if you noticed this but semantic coloring really over-uses orange for types in certain languages, such as Java (see image). This is out of sync with Github.com.
<img width="597" alt="image" src="https://user-images.githubusercontent.com/18719073/105387401-80b54800-5be3-11eb-8e60-826897b04cdc.png">

_Originally posted by @jeffgennari in https://github.com/primer/github-vscode-theme/issues/117#issuecomment-764809787_

**Comment by simurai** [comment]
@jeffgennari moved this to its own issue.

Mind linking to the above code? 🙇  Just in case it's public, of course.

**Comment by jeffgennari** [comment]
Of course, there is nothing special about the code, it is just a bunch of object declarations in Java. Again, this happens when semantic coloring is enabled. 

https://github.com/NationalSecurityAgency/ghidra/blob/2b08598dbabe6a9c911e712bd928e5f84fb00ee8/Ghidra/Features/Base/src/main/java/ghidra/util/state/ResultsState.java
